spec/services/encryption/encryptors/attribute_encryptor_spec.rb

```diff
@@ -8,8 +8,8 @@
 
   before do
     allow(AppConfig.env).to receive(:attribute_encryption_key).and_return(current_key)
-    allow(AppConfig.env).to receive(:attribute_encryption_key_queue).and_return(
-      [{ key: retired_key, cost: retired_cost }].to_json,
+    allow(IdentityConfig.store).to receive(:attribute_encryption_key_queue).and_return(
+      [{ 'key' => retired_key, 'cost' => retired_cost }],
```

Comment:
should we set `options: { symbolize_names: true }` on this one?

Reply:
The original config was a little misleading since the `to_json` and eventual JSON parse would return string keys: https://github.com/18F/identity-idp/pull/4847/files#diff-4ccf2eb5692dcb68d6fd1efef077acbc17ea0e48128580b0b4b02ea9197d42c6L19